api/Headers.json

```diff
@@ -0,0 +1,908 @@
+            "version_added": "29"
+          },
+          "safari": {
+            "version_added": "10.1"
+          },
+          "safari_ios": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "advanced_header_methods": {
+        "__compat": {
+          "description": "advanced Header methods",
+          "support": {
+            "webview_android": {
+              "version_added": true,
+              "notes": "<code>entries()</code>, <code>keys()</code>, <code>values()</code>, and support of <code>for...of</code>"
+            },
+            "chrome": {
+              "version_added": true,
+              "notes": "<code>entries()</code>, <code>keys()</code>, <code>values()</code>, and support of <code>for...of</code>"
+            },
+            "chrome_android": {
+              "version_added": true,
+              "notes": "<code>entries()</code>, <code>keys()</code>, <code>values()</code>, and support of <code>for...of</code>"
+            },
+            "edge": {
+              "version_added": "16",
+              "notes": "<code>entries()</code>, <code>keys()</code>, <code>values()</code>, and support of <code>for...of</code>"
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "44",
+              "notes": "<code>entries()</code>, <code>keys()</code>, <code>values()</code>, and support of <code>for...of</code>"
+            },
+            "firefox_android": {
+              "version_added": "44",
+              "notes": "<code>entries()</code>, <code>keys()</code>, <code>values()</code>, and support of <code>for...of</code>"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": true,
+              "notes": "<code>entries()</code>, <code>keys()</code>, <code>values()</code>, and support of <code>for...of</code>"
+            },
+            "opera_android": {
+              "version_added": true,
+              "notes": "<code>entries()</code>, <code>keys()</code>, <code>values()</code>, and support of <code>for...of</code>"
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          }
+        }
+      },
+      "lexicographical_sorting": {
+        "__compat": {
+          "description": "Lexicographical sorting, and values from duplicate header names combined when iterated.",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": "42"
+            },
+            "chrome_android": {
+              "version_added": "44"
```

Comment:
Where did you get the information about lexicographical sorting?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Headers Here but I made an error - they should be null according to the old compat table.